### ISSUE TITLE
write table to doc/docx

### ISSUE THREAD

**Issue by retsyo** [pyexcel/pyexcel#77]
first of all, I know excel is not word.  Then I apologize to say I this is actually not an issue but I don't know where to write it.

So, here my question is, is there a way to write a table into DOC or DOCX file?

Thanks

**Comment by chfw** [comment]
As you see, pyexcel does not support docx format. However, python has such [a library python-docx](https://python-docx.readthedocs.io/en/latest/) could do it and potentially can interface with pyexcel if a plugin is written.

**Comment by retsyo** [comment]
Yes, I know and use python-docx. However, for my current work, I write XLSX with openpyxl because I can check whether the result is right easily, but finnaly I have to fill the result as a table in MSWord file which is supplied by my leader. I can't copy the table from Excel to Word simplely because the format of table will changed since there are some merged cells.

Since pyexcel says 'Let you focus on data, instead of file formats', so maybe someday pyexcel can render table to DOC, PDF or PPT.  I only mean table since we all know that DOC, PDF or PPT can contain more than table

**Comment by chfw** [comment]
Thanks for your good wishes for pyexcel.

I see your use case that you would like to write data as a table in docx format, so that you could copy and paste it into one of your docx file. 

However, I thought the table format between MS products are common. Are there some special features in docx table and not in ms excel? Or are you using different office products?

**Comment by zoigo** [comment]
"_maybe someday pyexcel can render table to DOC, PDF or PPT_"
Quite a stretch, I doubt this idea is sane.
I used pyexcel to load data, and after some calculation I am outputting it in .docx table with help of docx-mailmerge.
(https://pypi.python.org/pypi/docx-mailmerge)
I used this fresh guide to get started: http://pbpython.com/python-word-template.html

The other day I also used to generate hundreds of fiddly tables to PDF using reportlab.
(https://pypi.python.org/pypi/reportlab)
